### PR TITLE
Part of #21421: Update FontFamilies.h, audio.h with new constant notation

### PR DESCRIPTION
**Pull request by jason-mills**

### CODE REVIEW

**Review by Gymnasiast — APPROVED** [review]

**Comment by Gymnasiast** [comment]
The ‘Lint commit message’ can be ignored if it is the only once, as I’m intending to squash this PR anyway.

**Review by tupaschoal — APPROVED** [review]

File: src/openrct2/interface/FontFamilies.h [hunk 1]
```diff
@@ -9,7 +9,7 @@
 
 #pragma once
 
-#define FAMILY_OPENRCT2_SPRITE nullptr
+constexpr std::nullptr_t kFamilyOpenRCT2Sprite = nullptr;
```
Comment (tupaschoal) [inline src/openrct2/interface/FontFamilies.h:12]:
```suggestion
#include <cstddef>

constexpr std::nullptr_t kFamilyOpenRCT2Sprite = nullptr;
```

**Review by tupaschoal — COMMENTED** [review]
Fix most builds, still need to understand why Android is not happy.

File: src/openrct2/interface/FontFamilies.h [hunk 2]
```diff
@@ -29,6 +31,6 @@ extern TTFontFamily const TTFFamilySansSerif;
 
 #else // NO_TTF
 
-#    define FAMILY(x) FAMILY_OPENRCT2_SPRITE
+#    define Family(x) kFamilyOpenRCT2Sprite
```
Comment (jason-mills) [inline src/openrct2/interface/FontFamilies.h:34]: Is the change here from FAMILY to Family breaking the define in Language.cpp

**Pull request merged**